code/modules/clothing/under/jobs/cargo.dm

```diff
@@ -53,7 +58,7 @@
 	wound = 10
 
 /obj/item/clothing/under/rank/cargo/miner/lavaland
-	desc = "A green uniform for operating in hazardous environments."
 	name = "shaft miner's jumpsuit"
+	desc = "A green uniform for operating in hazardous environments."
```

Comment:
since you've touched this i'm going to ask to rectify this trolling
```suggestion
	desc = "A grey uniform for operating in hazardous environments."
```
![изображение](https://github.com/tgstation/tgstation/assets/75863639/3424f026-d110-47d8-8a23-892278e9b0bb)

Reply:
What if I told you...
![image](https://github.com/tgstation/tgstation/assets/76465278/f4f672f8-a34a-4692-9620-6c56077d33d2)
its a green grey :trollface: